packages/test/test-version-utils/README.md

```diff
@@ -1,26 +1,137 @@
 # @fluidframework/test-version-utils
 
-Provide Fluid APIs in different version for testing.
-Separate into Loader/ContainerRuntime/DataObject+DDS category for now.
-- getLoaderApi
-- getContainerRuntimeApi
-- getDataRuntimeApi
+This is a package for writing and setting up Fluid end to end tests using `mocha` that will generate variants with
+a specific driver and different version combinations of Fluid API between layers via `TestObjectProvider` provided
+to the test. The different layers are loader, driver, container runtime and data runtime (includes DDS).  Version
+combinations and driver selection can be controlled via the command line for test set up using the provided
+ `describe*` functions.  For advanced usage, test suite can bypass the setup and command line and directly call our
```

Comment:
```suggestion
combinations and driver selection can be controlled via the `mocha` command line, assuming your test uses the provided
 `describe*` functions.  For advanced usage, a test can bypass this mechanism and directly call our
```

Reply:
Minor rewording, trying to be a little more explicit up front in "how" it works, everything made sense as I read it but I got to the end and didn't really get how you'd use this stuff.